### PR TITLE
Implement Koiki API client

### LINKED ISSUE #2: Create Koiki API client
This client needs to implement their only endpoint, `altaEnvios`

### PR DESCRIPTION
**Pull request by sauloperez**
Closes #2 

I went with a `Client` class that holds a method per endpoint, given Koiki's API has only an endpoint that we'll use. The other one is /ping which we don't need in production.

If we ever need to support new endpoints, we could then evolve this simple design into something like a class per endpoint and leaving `Client` to deal with `requests` only. So far, that's a complexity we don't need and we don't know if we'll ever need it.

Woocommerce API docs: https://woocommerce.github.io/woocommerce-rest-api-docs/?python#order-properties

### CODE REVIEW

File: koiki/sender.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    def to_dict(self):
+        return {
+            'nombreRemi': 'La Zona',
+            'direccionRemi': 'C/ La Zona, 1',
+            'codPostalRemi': '08186',
+            'poblacionRemi': 'Barcelona',
+            'provinciaRemi': 'Barcelona',
+            'paisRemi': 'ES',
+            'emailRemi': 'lazona@opcions.org',
+            'telefonoRemi': '+34518888191',
```
Comment (danypr92) [inline koiki/sender.py:14]: Can we move these content to a configuration file or envvars?

**Review by danypr92 — APPROVED** [review]

File: koiki/client.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import requests
+import logging
+
+from koiki.sender import Sender
+from koiki.recipient import Recipient
+
+HOST = 'https://rekistest.koiki.es/services'
```
[the remark this answers is not in the document]
Comment (sauloperez) [reply koiki/client.py:7]: yes, that's what I intend to do once Koiki gives the details of the production environment but I might as well do it now, yes.

File: koiki/sender.py (hunk 1 above)
Comment (sauloperez) [reply koiki/sender.py:14]: I don't know if that's the address we'll use. I need to validate it with Sergi.

**Comment by sauloperez** [comment]
Thanks for the review @danypr92 ! It was useful to get someone else to read this code :ok_hand:

**Pull request merged**